Makefile

```diff
@@ -56,6 +78,9 @@ server-dev:
 	DJANGO_SETTINGS_MODULE="pycon.dev_settings" DEBUG=True ./manage.py runserver 0:37266
 
 shell:
```

Comment:
maybe if we're adding DJANGO_SETTINGS_MODULE we should also  add `-dev` to the target, just like in the `server-` just to make sure nobody runs `make shell` on staging/production with wrong settings. :)

Reply:
Good point - being extra helpful definitely doesn't hurt :)